kebechet/overlays/stage/imagestreamtag.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     - name: latest
       from:
         kind: DockerImage
-        name: quay.io/thoth-station/kebechet-job:v1.0.6
+        name: quay.io/thoth-station/kebechet-job:v1.0.5
```

Comment:
shouldn't it be `v0.6.0-dev`?

Reply:
https://quay.io/repository/thoth-station/kebechet-job?tab=tags
the 1.0.5 image seemed to be working

Reply:
wasn't the one that was not allowing release of investigator?

Reply:
Yes, if there are no commits, we will release source-management and kebechet and bump to 1.0.7